webClient/src/app/dbbrowser-component.html

```diff
@@ -10,21 +10,22 @@
 -->
 
 <body style="background-color:ui-02">
-<label class = "title">
+<label class = "title" i18n="label|Label for apptitle@@apptitle-label">
 Database  Browser
 </label>  
  
 <div class ="large-window-padding">
  
   <div  class="row-in-flex margin-bottom-ten-px">
     <div class = "row-in-flex dropdown-background width-35">
-    <ibm-dropdown #connectionDropDown class="bx--dropdown" style="order:1" type="single" placeholder="Select a connection" size="lg" 
+    <ibm-dropdown #connectionDropDown class="bx--dropdown" style="order:1" type="single" placeholder="Select a connection"
+    i18n-placeholder="search placeholder|Placeholder for select conn@@select-conn-placeholder"     size="lg" 

```

Comment:
Why is this one "i18n-placeholder" when the rest are just "i18n"?

Reply:
It should be for translating an attribute of an object. See below from Angular. 
To mark an attribute for translation, add an attribute in the form of i18n-x, where x is the name of the attribute to translate. The following example shows how to mark the title attribute for translation by adding the i18n-title attribute on the img tag:

You also can assign a meaning, description, and id with the i18n-x="<meaning>|<description>@@<id>" syntax.